code-challenges.test.js

```diff
@@ -22,11 +25,34 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+// 1a) Pseudo code:
+// Create describe, it, and expect statements to describe the problem statement
+// Add appropriate inputs/outputs, with one expect line per input/output pair
+// Run the test on terminal (yarn jest) prior to creating the function to ensure it turns red (note: if set up correctly, should get an error saying the function is not defined)
 
+describe("oddity", () => {
+    it("a function that takes in a number input and evaulates whether it is even or odd", () => {
+      expect(oddity(num1)).toEqual("odd")
+      expect(oddity(num2)).toEqual("even")
+      expect(oddity(num3)).toEqual("odd")
+    })
+  })
```

Comment:
It doesn't look like you used the expected outcome. The prompt was not asking whether the numbers were even or odd.